cypress/integration/happy-path-with-accessibility.js

```diff
@@ -3,106 +3,114 @@ import 'cypress-axe';
 const specViolations = [];
 
 before(function () {
-    cy.fixture('postcodes.json').as('postcodes');
-    cy.log("Lets get started");
-    cy.visit('/');
-    Cypress.Cookies.defaults({
-        whitelist: '_get_help_to_retrain_session'
-    });
-
-    cy.contains('Accept cookies').click();
+  cy.fixture('postcodes.json').as('postcodes');
+  cy.log("Lets get started");
+  cy.visit('/');
+  Cypress.Cookies.defaults({
+    whitelist: ['_get_help_to_retrain_session', 'seen_cookie_message']
+  });
```

Comment:
Is this because of the cookies banner that prevents further interactions on the service?

Reply:
yea, the seen cookie message cookie was not maintained across different steps, so if we whitelist it it will be preserved